addons/clickable-quote-authors/addon.json

```
@@ -0,0 +1,38 @@
+{
+  "name": "Clickable forum quote authors ",
```

There's a trailing space.
```suggestion
  "name": "Clickable forum quote authors",
```